src/client/Microsoft.Identity.Client/ApiConfig/AbstractManagedIdentityAcquireTokenParameterBuilder.cs

```diff
@@ -20,15 +20,16 @@ namespace Microsoft.Identity.Client
     /// Abstract base class for managed identity application token request builders.
     /// </summary>
     /// <typeparam name="T"></typeparam>
-    public abstract class AbstractManagedIdentityParameterBuilder<T> : BaseAbstractAcquireTokenParameterBuilder<T>
+    public abstract class AbstractManagedIdentityAcquireTokenParameterBuilder<T> : BaseAbstractAcquireTokenParameterBuilder<T>
```

Comment:
Consider hiding this on mobile platforms.

Reply:
#if !SUPPORTS_CONFIDENTIAL_CLIENT
    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]  // hide managed identity flow on mobile
#endif